proposals/physical-cameras/README.md

```diff
@@ -0,0 +1,407 @@
+
+## Introduction
+
+This schema usefully bridges the description of physical cameras to USD scenes.
+
+@TODO PBRT has complex lens model, see RealisticCamera. Consider it for
+adoption here, perhaps as a secondary structure in Optics, or as a subclass.
+
+https://github.com/mmp/pbrt-v4/blob/master/src/pbrt/cameras.h
+
+PBRT defines Orthographic, Perspective, Spherical, and Realistic cameras.
+
+@TODO introduce at least Orthographic, in addtion to the present model
+which would be Perspective.
+
+## Use Cases
+
+### Interesting Looking Models
+
+The fx company has got a set of lenses in its
+library. These lenses are described by individual USD files that can be
+layered into a USD scene that records details of a shoot using those lenses.
+
+The camera and rig are composed of classes in this schema and are custom for
+every shot, but the lenses are reused between shots. Every morning the lenses
+are characterized, and if new values are recorded, they are added to the
+individual lens files as variants, and the current variant is set to latest.
+
+Before shots are archived, the variant in use is set appropriately, although
+during the shoot, the lens variant will typically be set to "current".
+
+### Likeable Puppets
+
+The stop motion animation company has got a large collection of camera bodies
+and lenses. These bodies and lenses travel around and are mixed and matched
+all day long during shoots on multiple sets, and when images are recorded,
+the combination for any given shot is constructed via layering those cameras
+and bodies, and associated with the image plane data, so that the plates and
+unique camera set ups always travel together.
+
+### Bellam Blvd Previs
+
+The previs company has got a camera kit they use in all their work. This kit
+consists of a single USD file with 8 variants, including Doll House, Over The
+Shoulder, and Out in the Desert. These variants are selected on a shot by shot
+basis, and characteristics of the individual lenses and bodies are overridden
+as needed.
+
+### Beloved Lens Co
+
+The Beloved Lens Co has sent us a box with it's latest offerings, Darling 16,
+Darling 24, a Gamin 8, and a Super Snoot 300. They've also kindly supplied USD
+files for these lenses, complete with both a custom lens model that properly
+describes the unique characteristics of their lenses, and also a best match
+PhysicalCameraLensModelPinhole for use in realtime previs scouting.
+
+### Smart Stage Studios
+
+insert LED wall scenario here
+
+### Streaming Behemoth
+
+Insert streamed camera metadata scenario here
+
+## Relationship to Image Plane proposal
+
+These should be complementary, and any information related to elements
+in this proposal, should be migrated to this proposal. Otherwise, the Image
+Plane proposal specifically describes the mapping of a capture image to an
+image plane, and the Physical Camera schema describes the mapping of objects
+in a scene to the image plane. So that's the pivot between them.
+
```

Comment:
Image plane proposal is here: [https://github.com/PixarAnimationStudios/OpenUSD-proposals/tree/main/proposals/image-planes](https://github.com/PixarAnimationStudios/OpenUSD-proposals/tree/main/proposals/image-planes)